guides/overview.md

```diff
@@ -4,9 +4,135 @@ Amber is designed to make common web-development tasks simple, and easy, maintai
+
+To install the Amber binary on Mac you can use
+
+```bash
+$ brew install amberframework/amber/amber
+```
+
+To verify that you have everything installed correctly, you should be able to run the following:
+
+```bash
+$ amber -v
+Amber CLI (amberframework.org) - vX.X.X
+```
+
+For other Linux distributions and guides please see Installation
+
+### Creating the Blog Application
+
+Amber comes with a number of scripts called generators that are designed to make your development life easier by creating everything that's necessary to start working on a particular task. One of these is the new application generator, which will provide you with the foundation of a fresh Amber application so that you don't have to write it yourself.
+
+To use this generator, open a terminal, navigate to a directory where you have rights to create files, and type:
+
+```bash
+$ amber new blog -d sqlite
+```
+
+This will create an Amber application called Blog in a `blog` directory using the sqlite database engine. If you don't specify a database the default will be `postgresql`
+
+Tip: You can see all of the command line options that the Rails application builder accepts by running `rails new -h`
+
+After you create the blog application, switch to its folder:
+
+```bash
+$ cd blog
+```
+
+You can then install your dependencies with this command
+
+```bash
+$ shards install
+```
+
+The `blog` directory has a number of auto-generated files and folders that make up the structure of an Amber application. Most of the work in this tutorial will happen in the `src` folder, but here's a basic rundown on the function of each of the files and folders that Rails created by default:
+
+| File Folder | Purpose |
+| :--- | :--- |
+| config/ | Contains configuration code for databases, environments, routing, and deployment |
+| db/ | Contains SQLite database and database migrations |
+| lib/ | Contains local installation of all dependent shards |
+| public/ | Location for html, css, and javscripts. This will be the default handler for static assets. |
+| spec/ | Location for application specification and tests |
+| src/ | Source code that composes your application |
+| .amber\_secret\_key | This file is used to encrypt application secrets so they are not stored in source control |
+| .gitignore | This file tells git which files and patterns it should ignore. |
+| docker-compose.yml | A file which defines services, networking, and drives for docker deployment. |
+| Dockerfile | A file which contains all the commands needed to build a docker image. |
+| README.md | A brief instruction manual about your application |
+
+## Hello, Amber!
+
+To begin with, let's get some text up on screen quickly. To do this, you need to get your Amber application server running
+
+### Starting up the Web Server
+
+You actually have a functional Rails application already. To see it, you need to start a web server on your development machine. You can do this by running the following in the`blog`directory:
```

Comment:
You actually have a functional Amber application already.

Based on @marksiemers comment here: https://github.com/amberframework/docs/pull/43/files#r154251839